### PR TITLE
Fix zimuku.org validation rules using anti-captcha

### PR DESCRIPTION
**Pull request by rxrw**
As mentioned before, #2106 is resolved by using anti-captcha provider.

### CODE REVIEW

**Comment by morpheus65535** [comment]
I'm not a fan of adding new dependencies to requirements.txt. It may break Bazarr startup in case the module isn't available for a platform or if compilation fails. That's the kind of stuff I thoroughly test in multiple environments.

There's no way to do it without pillow?

**Comment by rxrw** [comment]
> I'm not a fan of adding new dependencies to requirements.txt. It may break Bazarr startup in case the module isn't available for a platform or if compilation fails. That's the kind of stuff I thoroughly test in multiple environments.
> 
> There's no way to do it without pillow?

Zimuku's validation image is formatted in bmp, but anti-captcha only accept jpg or png. So pillow is required to convert it.

As an alternative option, I can use third-party online service to convert it. Is it acceptable?

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 setuptools
 lxml>=4.3.0
 numpy>=1.12.0
-webrtcvad-wheels>=2.0.10
\ No newline at end of file
+webrtcvad-wheels>=2.0.10
+Pillow==9.4.0
```
Comment (morpheus65535) [inline requirements.txt:5]: Any reason to have this specific version? Could it be `Pillow>=9.0.0` ?

**Comment by morpheus65535** [comment]
We'll go with the new requirement. Don't forget to add it to this: https://github.com/morpheus65535/bazarr/blob/development/bazarr/init.py#L79

**Comment by morpheus65535** [comment]
Something is failing while linting your PR. Please review the log.

File: requirements.txt (hunk 1 above)
Comment (rxrw) [reply requirements.txt]: I have changed the version specification. Pillow is just used for an image convert🤔

**Comment by morpheus65535** [comment]
I'll wait for hotio to get Pillow included in his images then merge this PR.

**Comment by kfstorm** [comment]
Does zimuku work now? I've set up anti-captcha and it works. But I still got this error:


Throttling zimuku for 10 minutes, until 23/05/07 11:51, because of: AttributeError. Exception info: "'NoneType' object has no attribute 'find'" | 07/05/2023 11:41:06
-- | --
Unexpected error in provider 'zimuku': Traceback (most recent call last): File "/app/bazarr/bin/bazarr/../libs/subliminal_patch/core.py", line 311, in list_subtitles_provider results = self[provider].list_subtitles(video, provider_languages) File "/app/bazarr/bin/bazarr/../libs/subliminal_patch/providers/zimuku.py", line 279, in list_subtitles for s in self.query( File "/app/bazarr/bin/bazarr/../libs/subliminal_patch/providers/zimuku.py", line 258, in query new_subs = self._parse_episode_page(episode_link, subs_year) File "/app/bazarr/bin/bazarr/../libs/subliminal_patch/providers/zimuku.py", line 167, in _parse_episode_page subs_body = bs_obj.find("div", class_="subs box clearfix").find("tbody")AttributeError: 'NoneType' object has no attribute 'find' | 07/05/2023 11:41:06

**Pull request merged**